lib/core/unittest/CMemoryUsageTest.cc

```diff
@@ -785,7 +786,7 @@ void CMemoryUsageTest::testDynamicSizeAlwaysZero() {
     CPPUNIT_ASSERT_EQUAL(haveStructPodCompilerSupport, test);
     test = core::memory_detail::SDynamicSizeAlwaysZero<std::pair<int, int>>::value();
     CPPUNIT_ASSERT_EQUAL(true, test);
-    test = boost::is_pod<SFoo>::value;
+    test = std::is_pod<SFoo>::value;
```

Comment:
You can now delete the comment on lines 762 to 770.

Also I imagine now `haveStructPodCompilerSupport` should always be `true`, so you could get rid of that too and replace with literal `true`s.

Reply:
done